source/js/storage.js

```diff
@@ -0,0 +1,179 @@
+const userInfo = {};
+const renderedText = {};
+
+window.addEventListener('DOMContentLoaded', init);
+
+// temporary init for demo
+async function init() {
+  getUserInfo().then(renderText);
+  bindButton();
+}
+
+/**
+ * After this function resolves, the userInfo object 
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8429666)
```suggestion
 * After this function resolves, the userInfo object
```